addon/components/item-picker/item-row/single/template.hbs

```diff
@@ -1,7 +1,7 @@
 <span class="pull-right item-picker-item-results-item-type">{{typeOfData}}</span>
 <a href="" {{action 'selectItem' model}}>
   <div class="item-picker-item-results-item-inner">
-    <h2 data-toggle="tooltip" title="{{model.title}}">
+    <h2 title="{{model.title}}">
```

Comment:
Same question here - this change will remove the bootstrap tooltip but cause most browsers to add a tooltip.